pages/landing/enterprise-eks/index.html

```diff
@@ -0,0 +1,191 @@
+
+how_it_works:
+  - heading: "Step 1: Choose your EKS configuration"
+    precode: |
+      <figure><figcaption>eks-configuration.yml</figcaption>
+        <pre>
+            <code class="language-yml">
+            # Configure CIS Benchmark compliance
+            cis:
+              enabled: true
+
+            # Choose from istio, consul-connect, aws-app-mesh
+            service_mesh: "istio"
+
+            # Choose from flux, jenkins, argocd
+            cicd: "flux"
+
+            # Choose from prometheus, datadog, cloudwatch
+            observability: prometheus
+
+            # Configure various common utilities
+            cluster_autoscaler:
+              enabled: true
+
+            sealed_secrets:
+              enabled: true
+
+            aws_load_balancer_controller:
+              enabled: true
+
+            external_dns:
+              enabled: true
+              domain: gruntwork.io
+            </code>
+        </pre>
+      </figure>
+    image_side: right
+    body: |
+      Pick and choose the technologies that you want included in your custom EKS cluster. We
+      support a wide range of popular configurations that include:
+      <ul>
+          <li>Service meshes (Istio, Consul connect, AWS App Mesh)</li>
+          <li>GitOps CI/CD (Flux, Jenkins, Argo)</li>
+          <li>UI dashboards (Porter, Grafana)</li>
+          <li>Observability (Prometheus, Datadog, CloudWatch, ELK)</li>
+          <li>And more (cluster-autoscaler, sealed secrets, AWS Load Balancer Controller, etc)</li>
+      </ul>
+      On top of that, all our enterprise-grade EKS clusters are hardened according to the <a
+      href="https://aws.amazon.com/blogs/containers/introducing-cis-amazon-eks-benchmark/">CIS Amazon EKS
+      Benchmark</a>, regardless of the chosen technologies.
+    logos:
+      - title: CIS
+        src: /assets/img/logos-compliance/cis-benchmark.png
+      - title: EKS
+        src: /assets/img/logos-technologies/logo-eks.png
+
+  - heading: "Step 2: Deploy your EKS cluster"
+    tab_code_preview: site.data.enterprise-landing-page-tabs-code
+    image_side: left
+    body: |
+      Deploy your custom configured EKS cluster. Our friendly user interface can generate code for your existing AWS
+      accounts and VPCs, or you can dive right in to the code. Or, use <a href="https://gruntwork.io/landing-zone-for-aws/">Gruntwork Landing Zone</a> and the
+      <a href="https://gruntwork.io/reference-architecture">Gruntwork Reference Architecture</a> for a complete,
+      end-to-end stack, built with extensibility in mind. No matter what you choose, everything is managed as code with Terraform.
+    logos:
+      - title: Terraform
+        src: /assets/img/landing-heroku-page/terraform-icon.png
+      - title: Docker
+        src: /assets/img/landing-heroku-page/docker-icon.png
+      - title: Packer
+        src: /assets/img/landing-heroku-page/packer-icon.png
+
+  - heading: "Step 3: Set up your app"
```

Comment:
```suggestion
  - heading: "Step 3: Configure your apps"
```

This has two benefits:

1. Repeats the "configure your EKS cluster" and "deploy your EKS cluster" structure.
2. Enterprises have many apps, not 1.